From 5f992b59a74cc6cde8fd20162a11065dc30fd7ab Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Fri, 26 Feb 2021 09:13:23 +0800
Subject: [PATCH] refpolicy-minimum: enable nscd_use_shm

Fixes:
avc: denied { listen } for pid=199 comm="systemd-resolve"
path="/run/systemd/resolve/io.systemd.Resolve"
scontext=system_u:system_r:systemd_resolved_t:s0
tcontext=system_u:system_r:systemd_resolved_t:s0
tclass=unix_stream_socket permissive=0

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/services/nscd.te | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy/modules/services/nscd.te b/policy/modules/services/nscd.te
index ada67edb1..9801fc228 100644
--- a/policy/modules/services/nscd.te
+++ b/policy/modules/services/nscd.te
@@ -15,7 +15,7 @@ gen_require(`
 ##	can use nscd shared memory.
 ##	</p>
 ## </desc>
-gen_tunable(nscd_use_shm, false)
+gen_tunable(nscd_use_shm, true)
 
 attribute_role nscd_roles;
 
-- 
2.17.1

